src/hotspot/share/compiler/compilerDirectives.cpp

```diff
@@ -434,11 +437,10 @@ DirectiveSet* DirectiveSet::compilecommand_compatibility_init(const methodHandle
       // Parse ccstr and create mask
```

Comment:
Could you update (or remove) this comment as part of the renaming?
Please also update the comment on line 430:
`    // Parse PrintIdealPhaseName and create an efficient lookup mask`

Reply:
Thanks, both updated now